app/Services/Arclight/ArclightService.php

```diff
@@ -43,18 +45,39 @@ public function doRequest(
      * Do a get request to the arclight API
      *
      * @param ResponseInterface $response
+     * @param bool $throw
      */
-    public function getContent(ResponseInterface $response)
+    public function getContent(ResponseInterface $response, bool $throw = true)
     {
-        $media_component = json_decode($response->getContent());
+        $media_component = json_decode($response->getContent($throw));
 
         if (isset($media_component->_embedded)) {
             return $media_component->_embedded;
         }
 
+        $status_code = $response->getStatusCode();
+
+        if ($status_code !== HttpResponse::HTTP_OK) {
```

Comment:
We should accept any 2xx value and not just 200.  If HTTP_OK is 200, then look for a HttpResponse value that works for any 2xx response